webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala

```diff
@@ -761,19 +762,16 @@ final case class PermissionsResponderADMLive(
 
         // get group
         groupIri <-
-          if (OntologyConstants.KnoraAdmin.BuiltInGroups.contains(createRequest.forGroup)) {
+          if (OntologyConstants.KnoraAdmin.BuiltInGroups.contains(createRequest.forGroup))
```

Comment:
**suggestion**: Lets keep our formatting aligned and always use curly braces to delimit `if` `else` blocks